tests/cursor/cursor-isDead-001.phpt

```diff
@@ -9,22 +9,25 @@ require_once __DIR__ . "/../utils/basic.inc";
 $manager = new MongoDB\Driver\Manager(STANDALONE);
 
 $bulk = new MongoDB\Driver\BulkWrite();
-$bulk->insert(array('_id' => 1));
-$bulk->insert(array('_id' => 2));
-$bulk->insert(array('_id' => 3));
+$bulk->insert(['_id' => 1]);
+$bulk->insert(['_id' => 2]);
+$bulk->insert(['_id' => 3]);
 $manager->executeBulkWrite(NS, $bulk);
 
-$cursor = $manager->executeQuery(NS, new MongoDB\Driver\Query(array(), array('batchSize' => 2)));
+$cursor = $manager->executeQuery(NS, new MongoDB\Driver\Query([], ['batchSize' => 2]));
 
 foreach ($cursor as $_) {
     var_dump($cursor->isDead());
 }
 
+var_dump($cursor->isDead());
+
 ?>
 ===DONE===
 <?php exit(0); ?>
 --EXPECT--
 bool(false)
 bool(false)
+bool(false)
```

Comment:
It's interesting to note that this same test would fail on PHPC 1.1.x. If we extend the test to insert 8 documents and query with a batch size of 3, we can see that `mongoc_cursor_is_alive()` starts return false once the 7th document is reached (i.e. we've entered the final batch, even though we still have more documents to iterate).